docs/GraFx-Studio/guides/connector-hub/index.md

```diff
@@ -0,0 +1,53 @@
+# Connector Hub
+
+## Intro
+
+The Connector Hub is the central repository where you can find connectors you can install into your environment.
+
+## Overview of installed connectors
+
+As an Environment Admin, navigate to:
+
+My Environments > [Your Environment] > Settings > Connectors
+
+![screenshot-full](ch01.png)
+
+Select the tab Connectors
+
+![screenshot-full](ch02.png)
+
+Overview of the installed connectors
+
+![screenshot-full](ch03.png)
+
+## Add a connector
+
+Navigate to the list of installed connectors (see above) and click "+ Add connector"
+
+A pop-up window will display.
+
+![screenshot-full](ch04.png)
```

Comment:
Mss meer uitleg bij Connector Hub screenshot:
- By ... (built by CHILI publish or 3rd party --> https://wonderful-sky-012b9bc03-399.westeurope.1.azurestaticapps.net/GraFx-Studio/concepts/connectors/: Type of Connectors) 
- Media - Data - ... (type connector)

Reply:
done